doc(rancher): suggest managed dhcp for creating guest cluster (#703)
harvester/docs: docs/rancher/node/k3s-cluster.md

:::note

- Harvester K3s node driver is in **Tech Preview**.

> - Provisioning K3s Kubernetes clusters requires a means to configure the IP address of the underlying virtual machines. A typical way is to use DHCP. If you don't have an existing DHCP server on the VLAN network where the virtual machines are attached to, or you don't bother to spin up a new DHCP server, you can use the [Managed DHCP](../../advanced/addons/managed-dhcp.md) feature to achieve the same goal.

Reviewer: ```suggestion
- Provisioning K3s Kubernetes clusters involves configuring the IP address of the underlying virtual machines. You can do this using a DHCP server on the VLAN network that the virtual machines are attached to. If such a server does not exist on the network, you can use the [Managed DHCP](../../advanced/addons/managed-dhcp.md) feature to configure the IP address.
```

Author: I've updated the wording. PTAL. Thank you!
